plugins/synapticsmst/fu-plugin-synapticsmst.c

```diff
@@ -392,3 +390,15 @@ fu_plugin_coldplug (FuPlugin *plugin, GError **error)
 		g_debug ("error enumerating");
 	return TRUE;
 }
+
+void
+fu_plugin_init (FuPlugin *plugin)
+{
+	FuPluginData *data = fu_plugin_alloc_data (plugin, sizeof (FuPluginData));
+
+	data->system_type = fu_plugin_get_dmi_value (plugin,
```

Comment:
same here I guess, although in both cases the data should always be valid...

Reply:
Since the return type was const it seemed like a waste of memory to dup, but if you'd prefer dup'ing I'll shift it around.